check_tv.py

```
@@ -344,15 +407,15 @@ def start_check(driver, live_url, haha):
             logging.info("--END-------------live_api-----------------")
             logging.info("wait for offine now... and start countdown")
             try:
-              offline_check(driver, live_url, live_spare_url, inport)
+              offline_check(driver, live_url, live_spare_url, inport, titleforgmail)
             except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)